apps/oxlint/src/lint/mod.rs

```diff
@@ -92,7 +92,7 @@ impl Runner for LintRunner {
         let cwd = std::env::current_dir().unwrap().into_boxed_path();
         let lint_options = LintOptions::default()
             .with_filter(filter)
-            .with_config_path(basic_options.config)
+            .with_config_path(basic_options.config().map(Into::into))
```

Comment:
I've looked into the LSP, I'm working on a PR updating our LSP to take advantage of this feature. Is it possible to provide a way to resolve the path in the `with_config_path` instead of doing it in the `basic_options`?

It allows me to pass in a `None` value here and let this logic resolve it.

https://github.com/oxc-project/oxc/blob/9c64b1296307cfc39972a20768c7c97b287469c9/crates/oxc_language_server/src/main.rs#L350

Reply:
Maybe adding a dedicated setter would be better, Or even providing a function that is both public and is used in the `basic_options` implementation so we can share that code.